tests/routes_tests/test_auth.py

```diff
@@ -103,51 +126,224 @@ def test_jurisdictionadmin_start(client: FlaskClient):
     assert rv.status_code == 302
 
 
-def test_jurisdictionadmin_callback(client: FlaskClient):
-    org = create_organization("Test Organization")
-    election = Election(
-        id=str(uuid.uuid4()),
-        audit_name="Test JA callback",
-        organization_id=org.id,
-        is_multi_jurisdiction=True,
-    )
-    jurisdiction = Jurisdiction(
-        election_id=election.id, id=str(uuid.uuid4()), name="Test Jurisdiction"
-    )
-    u = User(
-        id=str(uuid.uuid4()), email="bar@example.com", external_id="bar@example.com"
-    )
-    j_admin = JurisdictionAdministration(user_id=u.id, jurisdiction_id=jurisdiction.id)
-
-    db.session.add(election)
-    db.session.add(jurisdiction)
-    db.session.add(u)
-    db.session.add(j_admin)
-    db.session.commit()
-
```

Comment:
did the behavior change here where you're no longer expecting a user in the table coming from back Auth0? I think that's worth a longer discussion if so.

Reply:
I just removed this because the same code is now executed by the `jurisdiction_id` fixture (and the chain of fixtures it depends on).

Hopefully there's no behavior change. But maybe I missed something because I haven't actually worked with the auth0 integration yet.

Reply:
got it, so no change in behavior, just a different place that creates the local admin jurisdiction user? Ok great.